src/tokens/tokens.service.ts

```diff
@@ -122,15 +109,10 @@ export class TokensService {
     }
   }
 
-  private async getStream(ctx: Context) {
-    const stream = this.stream;
-    if (stream !== undefined) {
-      return stream;
-    }
+  private async getStream(ctx: Context, namespace: string) {
     await this.migrationCheck(ctx);
```

Comment:
Do we still need this migration check? I'm not sure how much value it's adding... might be worth evaluating if it can just be stripped out in this next release.

Reply:
Yeah I'm really not sure. I tried to not touch/break as much of the existing "migration" code as possible because I don't fully understand all the iterations it has gone through. But I'm happy to delete more code if it's safe.

Reply:
This was there to look for old event streams/subscriptions and give you a nudge in the logs - "hey looks like you may want to delete and clean up old streams".

Since we've now decided to forcefully delete old streams, it doesn't seem useful anymore. It's worth deciding if we need to look for and delete any of the older spellings, in addition to the current stream though.

Reply:
I thought some more about this, and I think we should definitely remove it.

I believe it should be part of the release notes - ie if you have an old token connector instance, check its startup logs to be sure there are no warnings logged at startup. If there are warnings logged about any event streams or subscriptions, you should address those warnings _before_ upgrading. The warnings will _not_ be logged after upgrading to this new version.